spec/changelog.yml

```diff
@@ -7,6 +7,9 @@
   - description: Prepare for next version
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/509
+  - description: Improve system benchmarks definition
```

Comment:
Could we describe here how the definition is being improved? It seems that relocates some files, could this be mentioned here?